modules/tinymce/src/core/main/ts/dom/StyleSheetLoader.ts

```diff
@@ -20,8 +22,8 @@ import Tools from '../api/util/Tools';
  */
 
 export interface StyleSheetLoader {
-  load: (url: string, loadedCallback: Function, errorCallback?: Function) => void;
-  loadAll: (urls: string[], success: Function, failure: Function) => void;
+  load: (styleContainer: StyleContainer, url: string, loadedCallback: Function, errorCallback?: Function) => void;
```

Comment:
This is an API breaking change, so it couldn't be done until 6.0.0. Additionally, it also uses a Sugar API which can't be done for pubic APIs.

If you change it to a passing an optional DomElement as the final argument then this might work.

Reply:
I just noticed this isn't in the `api` directory, however this is exposed to the public API via `DOMUtils.styleSheetLoader`, so it is technically in the wrong directory.

Reply:
There's a `document: Document` passed to the "constructors" of DOMUtils and StyleSheetLoader. I wonder if I could make that a `Document | ShadowRoot`.